dev/AutoSuggestBox/AutoSuggestBox.vcxitems

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
license header for all new added files.

Reply:
I'm not seeing that in other vcxitems. Can you point me to one that has what you're looking for?